passage/src/main/java/id/passage/android/PassageCurrentUser.kt

```diff
@@ -109,7 +48,7 @@ final class PassageUser private constructor(
      * @return MagicLink?
      * @throws PassageUserException
      */
-    public suspend fun changeEmail(newEmail: String): MagicLink {
+    suspend fun changeEmail(newEmail: String): MagicLink {
```

Comment:
Let's add language parameter here
```suggestion
    suspend fun changeEmail(newEmail: String, language: String? = null): MagicLink {
```